modules/installation/nav-installation-guide.adoc

```diff
@@ -21,7 +21,6 @@ endif::[]
 *** xref:network-requirements.adoc[Network Requirements]
 **** xref:ports.adoc[Required Network Ports]
 *** xref:client-requirements.adoc[Client Requirements]
-*** xref:pubcloud-requirements.adoc[Public Cloud Requirements]
```

Comment:
Please see my first comment: do not remove Public Cloud content from the Installation Guide. This is wrong. Please check with me before starting with this kind of changes. 

I am not going to comment on the rest of the PR because it's fundamentally wrong.